lib/ops/ops.go

```diff
@@ -698,6 +698,9 @@ type Updates interface {
 	// RotatePlanetConfig rotates planet configuration package for the server specified in the request
 	RotatePlanetConfig(RotatePlanetConfigRequest) (*RotatePackageResponse, error)
 
+	// RotateTeleportConfig rotates teleport configuration package for the server specified in the request
+	RotateTeleportConfig(RotateTeleportConfigRequest) (nodeConfig *RotatePackageResponse, err error)
```

Comment:
nit: nodeConfig and err variables in interface description I assume are not needed? Or nodeConfig seems like a sort of weird name for RotatePackageResponse.